### ISSUE TITLE
[Usage]: How to get the logits from classification models.

### ISSUE THREAD

**Issue by kxzxvbk** [vllm-project/vllm#18386]
### Your current environment

pip install vllm

### How would you like to use vllm

Hi! When I use vllm for classification, I find that the returned output is something like: ClassificationOutput(num_classes=5), which only contains the probabilities.
Now, I want to directly get the logits (i.e. vectors before softmax).
I think it sounds easy, but I cannot find a good way. Anyone can help me?

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by noooop** [comment]
try 

override_pooler_config = {"normalize": False}

**Comment by kxzxvbk** [comment]
Oh! I find the error.
This should be modified as: {"pooling_type": "LAST", "normalize": False, "softmax": False, "step_tag_id": None, "returned_token_ids": None}
And the result will be correct. Thanks for your help!
